risc0/build/src/lib.rs

```diff
@@ -45,13 +45,11 @@ impl Risc0Metadata {
 struct Risc0Method {
     name: String,
     elf_path: PathBuf,
+    cache_dir: PathBuf, // It's a bit questionable if this belongs here
 }
 
 impl Risc0Method {
-    fn make_method_id<P>(&self, method_id_path: P)
-    where
-        P: AsRef<Path>,
-    {
+    fn make_method_id(&self) -> [u8; 384] {
```

Comment:
@flaub see here (and in a couple other places). I mentioned this in a previous version of the PR but i want to call it out again. There is currently no way to get a constexpr over the cxx bridge, so it either has to be redefined on the rust side or the rust side has to treat it as a [unsized] slice. Here I've got the actual number scattered all over but, if kept, I would collapse it into a constant somewhere. I've noticed a couple other constants duplicated in both rust and c++, so I don't think it'd be setting a bad precedent or anything. It's also unlikely to stick around for very long.

Reply:
Yeah it's probably fine for small stuff like this.